docs/USAGE-GUIDE.md

```diff
@@ -589,8 +589,9 @@ default is for s2n to accept all RSA Certs on the client side, and deny all cert
 ### verify_cert_trust_chain_fn
 
 ```c
-int verify_cert_trust_chain(uint8_t *der_cert_chain_in, uint32_t cert_chain_len, struct s2n_cert_public_key *public_key_out, void *context);
+int verify_cert_trust_chain(struct s2n_connection *conn, uint8_t *der_cert_chain_in, uint32_t cert_chain_len, struct s2n_cert_public_key *public_key_out, void *context);
 ```
+ - **conn** The connection certificate chain is validated for
```

Comment:
s/connection certificate/connection the certificate/

Reply:
Sure, fixed!